File: 1 Lista Slide 2/EX8.py
n1=int(input("Digite um número: "))
n2=int(input("Digite outro número: "))
if n1>n2:
    maior=n1
    menor=n2
else:
    maior=n2
    menor=n1
for i in range(menor+1,maior):
    print(i)

File: 1 Lista Slide 2/EX8.c
#include <stdio.h>

int main (void){
    int n1,n2;
    int maior,menor;
    scanf("%d", &n1);
    scanf("%d", &n2);
    if(n1>n2){
        maior=n1;
        menor=n2;
    }
    else{
        maior=n2;
        menor=n1;
    }
    printf("\n");
    while(menor+1<maior){
        printf("%d\n", menor+1);
        menor=menor+1;
    }
    return 0;
}